vspace/src/main/java/edu/asu/diging/vspace/web/staff/SpaceBlockController.java

```diff
@@ -0,0 +1,33 @@
+package edu.asu.diging.vspace.web.staff;
+
+import java.io.IOException;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+
+import edu.asu.diging.vspace.core.model.ISpace;
+import edu.asu.diging.vspace.core.model.ISpaceBlock;
+import edu.asu.diging.vspace.core.services.IContentBlockManager;
+
+@Controller
+public class SpaceBlockController {
+    
+    @Autowired
+    private IContentBlockManager contentBlockManager;
+    
+    @RequestMapping(value = "/staff/module/{moduleId}/slide/{id}/space/{blockId}", method = RequestMethod.GET)
+    public ResponseEntity<ISpaceBlock> getSpaceBlockSpace(@PathVariable("id") String slideId,
```

Comment:
this is returning a space block, right? not the space of a space block? so the method should be called `getSpaceBlock`